client/src/components/Jobs/Jobs.scss

```diff
@@ -59,10 +59,6 @@
   }
 }
 
-.job__path {
-  direction: rtl; /* move overflow/ellipsis to left of text */
```

Comment:
i feel the following came from Design: 
_overflow/ellipsis to left of text_  
as either a developer or a gave feedback that if you had the ellipses on end of the text then the text always looks the same.

Reply:
It did - though we need to find a different solution than the `direction: rtl` css property, as that property for some reason moves special characters to the left of the string. So a path like `frontera.home.sal//` would show as `//frontera.home.sal`, which is a little off. There is [this ticket](https://jira.tacc.utexas.edu/browse/FP-425) that is supposed to address this.

@nathanfranklin Think we should leave the weird special character formatting bug for now, and fix this later?

Reply:
>  as that property for some reason moves special characters to the left of the string

Yikes.  

> Think we should leave the weird special character formatting bug for now, and fix this later?

either way sounds good to me so will add approval now.  (👍  on ticket)